<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSpecialProgessBar</class>
 <widget class="QWidget" name="QSpecialProgessBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSpecialProgessBar</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Title_Label">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="fonts" stdset="0">
        <string notr="true">14</string>
       </property>
       <property name="margins" stdset="0">
        <string notr="true">top_4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="BarData_Label">
       <property name="styleSheet">
        <string notr="true">color: #fa9016;
font-size: 12px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="fonts" stdset="0">
        <string notr="true">12</string>
       </property>
       <property name="margins" stdset="0">
        <string notr="true">top_4</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
